packages/core/src/mpi/__tests__/match-patient.test.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  jaroWinklerSimilarity,
+  matchingPersonalIdentifiersRule,
+  matchingContactDetailsRule,
+} from "../match-patients";
+import { PatientMPI } from "../shared";
+import { testPatientMPI } from "./test_data";
+
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+
```

Comment:
I use null assertions here since the data is from the test file and its just a test